src/Search.js

```diff
@@ -10,15 +12,11 @@ const Search = () => {
         <div className="col">
           <form className="form-group search-box">
             <label htmlFor="customerName">Customer name</label>
-            <div className="search-row">
-              <input
-                type="text"
-                id="customerName"
-                className="form-control"
-                placeholder="Customer name"
-              />
-              <button className="btn btn-primary">Search</button>
-            </div>
+            <label className="searchIdLabel" htmlFor="customerName">Customer id</label>
```

Comment:
These `label` elements relate to the `input` elements in the `SearchBtn` component. This is a little strange - ideally components don't need to know about the internals of each other, but right now they do - these `label` elements know what the `id` attributes of things in the `SearchBtn` component are.

I would recommend moving the `input` elements back into this component, and having the `SearchBtn` just be the button. This has the advantage of making `SearchBtn` also be accurately named (Right now it is more than just a button!)

Additionally, if you wanted to remove some duplication, we could extract a new component, `SearchBox` which has as children a `label`, and `input`, and a `SearchBtn`.

Why don't you give that a go, it'll be good practice at extracting components :)